src/app/issuer/components/issuer-create/issuer-create.component.html

```diff
@@ -1,149 +1,160 @@
-
-		<div class="u-margin-top3x">
-			<form>
-				<altcha-widget auto="onfocus" id="altcha"></altcha-widget>
-			</form>
+		<div class="oeb-headline-container-sm">
+			<h1 hlmH1 class="tw-text-purple">{{ 'Issuer.createInstitution' | translate }}</h1>
+			<h1 hlmH1 class="tw-text-oebblack md:tw-text-xl tw-text-sm">{{ 'Issuer.awardBadeges' | translate }}</h1>
 		</div>
 
-		<label
-			[class.checkbox-is-error]="
-				issuerForm.rawControlMap.agreedTerms.dirty && !issuerForm.rawControlMap.agreedTerms.valid
-			"
-			*ngIf="dataProcessorUrl"
-			class="checkbox l-marginBottom-2x"
-			for="terms"
-		>
-			<input name="terms" id="terms" type="checkbox" [formControl]="issuerForm.rawControlMap.agreedTerms" />
-			<span class="checkbox-x-text"
-				>Ich habe die <a target="_blank" [href]="dataProcessorUrl">Datenschutzrichtlinien</a> gelesen und stimme
-				ihnen zu.
-				<div
-					*ngIf="issuerForm.rawControlMap.agreedTerms.dirty && !issuerForm.rawControlMap.agreedTerms.valid"
-					class="checkbox-x-errortext"
-				>
-					{{ 'Issuer.readPolicy' | translate }}
-				</div>
-			</span>
-		</label>
-
-		<div class="l-flex l-flex-1x u-margin-bottom3x u-margin-top4x">
-			<a [routerLink]="['/issuer']" class="button button-secondary" [disabled-when-requesting]="true">{{
-				'General.cancel' | translate
-			}}</a>
-			<button
-				type="submit"
-				class="button"
-				[disabled]="!!addIssuerFinished"
-				[loading-promises]="[addIssuerFinished]"
-				loading-message="Hinzufügen"
+		<form (ngSubmit)="onSubmit()" class="u-width-formsmall u-margin-top3x" novalidate>
+			<bg-formfield-image
+				#imageField
+				label="{{ 'Issuer.imageOptional' | translate }}"
+				imageLoaderName="issuer"
+				[placeholderImage]="issuerImagePlacholderUrl"
+				[control]="issuerForm.rawControlMap.issuer_image"
+			></bg-formfield-image>
+
+			<oeb-input
+				label="{{ 'General.name' | translate }}"
+				[control]="issuerForm.rawControlMap.issuer_name"
+				[fieldType]="'text'"
+				[autofocus]="true"
+				errorMessage="{{ 'Issuer.enterName' | translate }}"
+				#issuer_name_field
+			></oeb-input>
+
+			<oeb-input
+				[label]="'Website URL'"
+				[control]="issuerForm.rawControlMap.issuer_url"
+				[urlField]="true"
+				[fieldType]="'url'"
+				errorMessage="{{ 'Issuer.enterURL' | translate }}"
+				#issuer_url_field
+			></oeb-input>
+
+			<bg-formfield-select
+				class="u-margin-bottom2x"
+				[control]="issuerForm.rawControlMap.issuer_email"
+				label="{{ 'General.contact' | translate }} E-Mail"
+				placeholder="{{ 'Issuer.enterValidEmail' | translate }}"
+				[options]="emailsOptions"
+				errorMessage="{{ 'Issuer.enterValidEmail' | translate }}"
+			></bg-formfield-select>
+
+			<oeb-input
+				label="{{ 'General.description' | translate }}"
+				[control]="issuerForm.rawControlMap.issuer_description"
+				[fieldType]="'textarea'"
+				errorMessage="{{ 'Issuer.enterDescription' | translate }}"
+				#issuer_description_field
+			></oeb-input>
+
+			<bg-formfield-select
```

Comment:
the select fields dont have padding. And also break the page by being too long.

Reply:
I added a space above address.